src/types/mod.rs

```diff
@@ -36,6 +36,10 @@ mod geometry;
 
 pub use geometry::Geometry;
 
+mod link;
+
+pub use link::{Icon as LinkTypeIcon, Link as LinkTypeLink, RefreshMode, ViewRefreshMode};
```

Comment:
Not a fan of these aliases, but I couldn't think of anything better. Decided to alias `Link` only because `Icon` was aliased. WDYT?

Reply:
I think I'd prefer we only alias `Icon` to `LinkTypeIcon` and remove the `LinkTypeLink` to keep things as consistent with the spec as possible